commit 0c548bab17c2ecd457a84c78ca53a10f389b316e
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Thu Jan 24 14:45:03 2019 +0200

    samples: mei: use /dev/mei0 instead of /dev/mei
    
    commit c4a46acf1db3ce547d290c29e55b3476c78dd76c upstream.
    
    The device was moved from misc device to character devices
    to support multiple mei devices.
    
    Cc: <stable@vger.kernel.org> #v4.9+
    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/samples/mei/mei-amt-version.c b/samples/mei/mei-amt-version.c
index 57d0d871dcf7..bb9988914a56 100644
--- a/samples/mei/mei-amt-version.c
+++ b/samples/mei/mei-amt-version.c
@@ -117,7 +117,7 @@ static bool mei_init(struct mei *me, const uuid_le *guid,
 
 	me->verbose = verbose;
 
-	me->fd = open("/dev/mei", O_RDWR);
+	me->fd = open("/dev/mei0", O_RDWR);
 	if (me->fd == -1) {
 		mei_err(me, "Cannot establish a handle to the Intel MEI driver\n");
 		goto err;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1321_0c548bab17c2ecd457a84c78ca53a10f389b316e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1321_0c548bab17c2ecd457a84c78ca53a10f389b316e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
